packages/spacecat-shared-data-access/test/unit/models/site/audit-config-type.test.js

```diff
@@ -118,4 +118,20 @@ describe('AuditConfigType Tests', () => {
       expect(dynamoItem.excludedURLs).to.eql(urls);
     });
   });
+
+  describe('updateManualOverwrites Method', () => {
+    it('updates the manualOverwrites array', () => {
+      const auditConfigType = AuditConfigType();
+      const newManualOverwrites = ['brokenTargetURL', 'targetURL'];
```

Comment:
See the comment above, we need to pass both the URL to overwrite with and the URL that is broken and should be overwritten, in pairs.